api/repositories/service_binding_repository_test.go

```diff
@@ -840,6 +930,9 @@ var _ = Describe("ServiceBindingRepo", func() {
 					Expect(k8s.PatchResource(ctx, k8sClient, serviceBinding3, func() {
 						serviceBinding3.Labels = map[string]string{"not_foo": "NOT_FOO"}
 					})).To(Succeed())
+					Expect(k8s.PatchResource(ctx, k8sClient, serviceBinding4, func() {
```

Comment:
This context is all about filtering by label selector. Does the new binding need to have those labels set? If not, then we would minimise the diff as tests within this context would not change